input/pagecontent/use-cases.md

```diff
@@ -10,39 +13,34 @@ This page contains the use cases for AU eRequesting for the first release of the
   <tr>
     <th style="width:5%">ID</th>
     <th style="width:45%">Description</th>
-    <th style="width:17%">Actors</th>
-    <th style="width:17%">Systems</th>
-    <th style="width:18%">Data Groups</th>
+    <th style="width:17%">Actor(s)</th>
+    <th style="width:18%">AUeReqDI/AUCDI Data Group(s)</th>
   </tr>
 </thead>
 <tbody>
   <tr>
     <td>D01</td>
     <td><strong>Request for diagnostic imaging (generic data collection requirements)</strong><br><br>A requesting clinician requires diagnostic imaging for a patient, following a consultation about an affected body part. The request contains information such as: Patient demographics (including sex and gender), Targeted body part, Type of imaging required, Reason for requesting imaging, relevant clinical history, any precautions about the patient’s other conditions such as device implants/allergies/medicines/related pathology results, destination(s) where the results need to be sent back to (including Copy To/Carbon Copy/copy to practitioners, as well as the original requestor), and information to support billing.</td>
-    <td>Patient<br/>Requesting Practitioner (e.g. GP)</td>
-    <td>Requesting System</td>
-    <td>Patient Details<br>Requestor Details<br>Requested Service Details<br>Relevant Clinical Details<br>Billing Details<br>Preferred Reporting Clinician</td>
+    <td><a href="ActorDefinition-au-erequesting-actor-placer.html">AU eRequesting Placer actor</a></td>
+    <td>Service request<br>Medical imaging request<br>Implanted medical device summary<br>Adverse reaction risk summary<br>Problem/Diagnosis summary<br>Sex and gender summary<br></td>
```

Comment:
Bullet point the list of Data Groups. Apply to all rows

Reply:
Done